packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -13,11 +13,13 @@ $ds-c-inset-border-width: $spacer-half;
 
   // Choice label
   + label {
-    align-items: center;
+    align-items: flex-start;
```

Comment:
Is this necessary? I removed `align-items: flex-start;` and `flex-wrap: wrap;` and everything worked fine still